middleware_test.go

```diff
@@ -22,14 +23,21 @@ import (
 	"github.com/acronis/go-authkit/jwt"
 )
 
+const (
+	errDomain       = "TestDomain"
+	testBearerToken = "a.b.c"
+)
+
 type mockJWTAuthMiddlewareNextHandler struct {
+	ctx       context.Context
 	called    int
 	jwtClaims jwt.Claims
```

Comment:
There’s no need to have `jwtClaims` now since it can be derived from the `ctx`.

I suggest replacing the `ctx` field with a `lastRequest` field for the following reasons:
a) `lastRequest` is clearer and more intuitive to understand in this context.
b) Storing `ctx` in a struct is generally [considered](https://go.dev/blog/context-and-structs) a poor practice. It may confuse other developers.